packages/documentation/src/stories/utilities/flex/flex.stories.ts

```diff
@@ -0,0 +1,143 @@
+  render: () =>
+    html`
+      <div class="d-flex align-items-end">
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray px-8 py-48">Content</div>
+      </div>
+    `,
+};
+
+export const AlignSelf: Story = {
+  render: () =>
+    html`
+      <div class="d-flex">
+        <div class="bg-gray px-8 py-48">Content</div>
+        <div class="bg-yellow align-self-end p-8">Content</div>
+        <div class="bg-gray px-8 py-48">Content</div>
+      </div>
+    `,
+};
+
+export const JustifyContent: Story = {
+  render: () =>
+    html`
+      <div class="d-flex justify-content-between">
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+      </div>
+    `,
+};
+
+export const GrowShrink: Story = {
+  render: () =>
+    html`
+      <div class="d-flex">
+        <div class="bg-yellow flex-grow-1 p-8">I'm growing</div>
+        <div class="bg-gray flex-shrink-1 p-8">I'm shrinking</div>
+      </div>
+    `,
+};
+
+export const AlignContent: Story = {
+  render: () =>
+    html`
+      <div style="height: 14rem" class="d-flex flex-wrap align-content-around">
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+      </div>
+    `,
+};
+
+export const Order: Story = {
+  render: () =>
+    html`
+      <div class="d-flex">
+        <div class="bg-yellow p-8 order-3">First element, positioned last</div>
+        <div class="bg-gray p-8 order-2">Second element</div>
+        <div class="bg-yellow p-8 order-1">Last element, positioned first</div>
+      </div>
+    `,
+};
+
+export const Wrap: Story = {
+  render: () =>
+    html`
+      <div class="d-flex flex-wrap">
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
+        <div class="bg-yellow p-8">Content</div>
+        <div class="bg-gray p-8">Content</div>
```

Comment:
You could show the index here to make wrap-reverse easier to understand:
```suggestion
        ${Array.from({length: 14}).map((_, i) => html`
          <div class="${i % 2 === 0 ? 'bg-gray' : 'bg-yellow'} p-8">Content ${i + 1}</div>
        `)}
```